api/controllers/OpenEAppController.js

```diff
@@ -150,6 +151,42 @@ const OpenEAppController = {
 
         return expired;
     },
+
+    async downloadReceipt(req, res) {
+        try {
+            await OpenEAppController._streamReceiptToClient(req, res);
+        } catch (err) {
+            sails.log.error(err);
+            return res.serverError();
+        }
+    },
+
+    async _streamReceiptToClient(req, res) {
```

Comment:
do you have a few streams you have to handle? (you can leave it for now - but might be worth considering a helper function that handles streams generically)

Reply:
this is the second one, AFAIK there shouldn't be anymore but you never know 🤷

Reply:
Good point. I will make that change 👍